exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -54,6 +58,14 @@ func (si *signingRoundTripper) RoundTrip(req *http.Request) (*http.Response, err
 
 	body := bytes.NewReader(content)
 
+	// Add the sdk and system information to the User-Agent header of the request
+	FormatHeaderValues(req, aws.SDKName, aws.SDKVersion, runtime.Version(), runtime.GOOS, runtime.GOARCH)
+
+	v := os.Getenv("AWS_EXECUTION_ENV")
```

Comment:
Let's make the AWS_EXECUTION_ENV an extra.

extras := []string{runtime.GOOS, runtime.GOARCH)}
if v := os.Getenv("AWS_EXECUTION_ENV"); v != nil {
    extras = append(extras, v)
}
addUserAgent(req, aws.SDKName, aws.SDKVersion, runtime.Version(), extras...)

Reply:
Sure, I will make this change